### ISSUE TITLE
False positive cascade_invocations

### ISSUE THREAD

**Issue by lexaknyazev** [dart-lang/linter#1023]
Dart SDK 2.0.0-dev.61.0

```dart
import 'dart:html';

void main() {
  final _foo = querySelector('#foo');
  final _bar = querySelector('#bar');

  _foo.style.width = '1px';
  _bar.style.height = '1px'; // <--- cascade_invocations
}
```

**Comment by apaatsio** [comment]
I can't reproduce this. I believe this is the same bug as #1323 which has already been fixed.

```
❯ dartanalyzer --version
dartanalyzer version 2.2.0-edge.0a7dcf17eb5f2450480527d6ad1e201fb47f1e36
❯ dart --version
Dart VM version: 2.1.2-dev.0.0.flutter-0a7dcf17eb (Tue Feb 12 01:59:15 2019 +0000) on "linux_x64"
```

**Comment by pq** [comment]
Thanks @apaatsio!  This looks fixed and covered by https://github.com/dart-lang/linter/pull/1328.  Cheers!
